Eastern-Research-Group/mywaterway
# Feature/128 add summarize param to waterbody report

## Related Issues:
* https://jira.epa.gov/browse/HMW-128

## Main Changes:
* Updated the `actions` service calls, on the waterbody report page, to use the `summarize=Y` parameter.
  * This change is going to display different results in the `Plans to Restore Water Quality` section. In the original code we were looking at the `specificWaters` value, which is only available from the `actions` service if you don't use the `summarize=Y` parameter. What we were doing was going through each action, finding the waters with the same assessment unit identifier, and then building a list of parameters from that. Basically, we were figuring out what parameters in the action affect the waterbody you are viewing. After this change we will just be displaying all parameters that are part of the action, which could include parameters for other waterbodies. To see what I'm talking about visit http://localhost:3000/waterbody-report/DOEE/DCANA00E_02/2020 and https://mywaterway.epa.gov/waterbody-report/DOEE/DCANA00E_02/2020. You will see that in production there are a lot of actions that say "No impairments found". I think this is ok since that table doesn't say anything about the impairments being specific to the waterbody being viewed. 
* Fixed a styling issue where extra padding was being added to the top of the accordions in the `What is this water used for?` section. This was caused by switching the Accordion titles from `span` to `p`. The waterbody report page uses `boxSectionStyles` which has some styles for `p` which is where that extra padding came from. 

## Steps To Test:
1. Navigate to http://localhost:3000/waterbody-report/11113300/NHLAK600020803-08/2018 and keep track of how long it takes to load
2. Verify the load time is under 10 seconds 
    * In production this takes about 30 seconds. 
3. Verify everything loads in correctly
4. Verify there is no top margin in the `What is this water used for?` section
